app/controllers/item.js

```diff
@@ -31,134 +32,141 @@ var UserSourceAuth = require('app/models/userSourceAuth');
 var UserStorageAuth = require('app/models/userStorageAuth');
 var validateParams = require('app/lib/validateParams');
 
+var util= require('util');
+
 var queue = kue.createQueue();
 
-queue.process('storeItemData', function(queueJob, done) {
-  debug('process queueJob %s', queueJob.id);
+var tools = require('app/lib/utils/debuggingTools');
```

Comment:
requires should all take place at the start of the module in alphabetical order unless there's a particular need to place them elsewhere in the module.